README.md

```diff
@@ -39,13 +39,13 @@ it('should be allow to check if promise rejected', function() {
 
 ## .rejectedWith
```

Comment:
Suggestion, `.rejectedWith(err)` to match the `.fulfilledWith(value)` change above?
Thanks